utilities.tex

```diff
@@ -310,7 +309,7 @@
 \end{itemdecl}
 \begin{itemdescr}
 \pnum
-\effects Equivalent to \tcode{\textit{INVOKE}(std::forward<F>(f), std::forward<Args>(args)...)}~(\cxxref{func.require}).
+\effects Equivalent to: \\ \tcode{return \textit{INVOKE}(std::forward<F>(f), std::forward<Args>(args)...);} (\cxxref{func.require}).
```

Comment:
I think it reads slightly better with the `return` statement and the cross-reference on the same line.